packages/cli/src/sso.ee/saml/routes/saml.controller.ee.ts

```diff
@@ -1,15 +1,14 @@
-import { validate } from 'class-validator';
-import express from 'express';
+import { SamlAcsDto, SamlPreferences, SamlToggleDto } from '@n8n/api-types';
+import { Response } from 'express';
```

Comment:
This should probably be
```suggestion
import type { Response } from 'express';
```

Reply:
we can't do that because then eslint starts complaining about this being used by decorator metadata, even if we `Response` isn't a runtime type and even if we don't actually use this type.
I plan to clean up this code in the future to stop using raw request and response objects. we can then remove this import.